### PR TITLE
3.7.x: Porting improvements from #11225 to beez

### PR DESCRIPTION
**Pull request by infograf768**
See test instructions in #11225

After patch, once logged in frontend displaying an article list, if you have access to an article and edit/create, you should get.

screen shot 2016-10-01 at 12 17 59

@alikon 
Please check/test

### CODE REVIEW

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on b21a9a439d724c2d3debf88e92ef5fe06b6d9bdd<br /><br />Works here :smile: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12240">issues.joomla.org/joomla-cms/12240</a>.</sub>

File: templates/beez3/html/com_content/category/default_articles.php [hunk 1]
```diff
@@ -14,10 +14,25 @@
 JHtml::addIncludePath(JPATH_COMPONENT.'/helpers/html');
 JHtml::_('behavior.framework');
 
-$n = count($this->items);
+$params    = &$this->item->params;
```
Comment (wilsonge) [inline templates/beez3/html/com_content/category/default_articles.php]: Does this need to be by reference?

Comment (infograf768) [inline templates/beez3/html/com_content/category/default_articles.php]: This is what we have in core. See la/joomla-cms/blob/3.7.x/components/com_content/views/category/tmpl/default_articles.php#L15

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on b21a9a439d724c2d3debf88e92ef5fe06b6d9bdd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12240">issues.joomla.org/joomla-cms/12240</a>.</sub>

**Comment by infograf768** [comment]
@wilsonge 
i suggest to merge this and check later for this piece of code both for core and beez

**Pull request merged**